web/yo/app/scripts/controllers/drugs.js

```diff
@@ -71,7 +75,7 @@ angular.module('oncokbApp')
 
             $scope.removeDrug = function (drug) {
                 if ($scope.drugMap[drug.uuid]) {
-                    var genes = _.map($scope.drugMap[drug.uuid], (gene) => {
+                    var genes = _.map($scope.drugMap[drug.uuid], function(gene){
```

Comment:
I think you can code in this way:
var genes = _.map($scope.drugMap[drug.uuid], 'geneName');
https://lodash.com/docs/#map